### PR TITLE
[deployment] Kubernetes Manifests

### PR DESCRIPTION
**Pull request by daviddyball**
## Hey, I just made a Pull Request!

Add some Kubernetes YAML manifests for deploying Backstage

TODO: Make it a proper Helm chart

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [x] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: install/kubernetes/app.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: backstage
+  labes:
```
Comment (freben) [inline install/kubernetes/app.yaml:5]: labels :)

File: install/kubernetes/backend.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: backstage-backend
+  labes:
```
Comment (freben) [inline install/kubernetes/backend.yaml:5]: labels

File: install/kubernetes/backend.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      app: backstage
+      component: backend
+    template:
+      metadata:
+        labels:
+          app: backstage
+          component: backend
+      spec:
+        containers:
+        - name: app
```
Comment (freben) [inline install/kubernetes/backend.yaml:21]: backend perhaps

**Review by freben — APPROVED** [review]
Awesome! Two small spelling mistakes and then looking good as far as I know at least :)

**Comment by daviddyball** [comment]
@freben good spot. That's what I get for not doing a `kubectl apply`.

**Comment by freben** [comment]
Thank you! 👌

**Pull request merged**